src/PKSim.Presentation/Presenters/Simulations/SimulationCompoundInteractionSelectionPresenter.cs

```diff
@@ -85,12 +85,20 @@ private SimulationInteractionProcessSelectionDTO mapFrom(SimulationPartialProces
       public void CompoundProcessSelectionChanged(SimulationInteractionProcessSelectionDTO dto, PartialProcess interactionProcess)
       {
          if (processAlreadySelected(interactionProcess))
+         {
+            refreshView();
             throw new CannotSelectThePartialProcessMoreThanOnceException(interactionProcess);
+         }
 
          dto.SimulationPartialProcess.CompoundProcess = interactionProcess;
          updateWaring();
       }
 
+      private void refreshView()
+      {
+         _view.Repaint();
```

Comment:
Probably required after DevExpress update. We need to refresh the view after the exception is thrown